modules/nw-networkpolicy-create-cli.adoc

```diff
@@ -116,6 +117,127 @@ spec:
 ifdef::multi[]
 +
 --
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+.Allow ingress traffic to one pod from a particular namespace
++
+This policy allows traffic to pods labelled `pod-a` from pods running in `namespace-y`.
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: allow-traffic-pod
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector:
+   matchLabels:
+      pod: pod-a
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+           kubernetes.io/metadata.name: namespace-y
+----
+ifdef::multi[]
++
+--
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+ifdef::multi[]
+.Restrict traffic to a service
++
+This policy blocks traffic to pods without the `app=bookstore` label. Traffic to pods with the `app=bookstore` label is allowed. Your application could be a REST API server, marked with labels `app=bookstore` and `role=api`. 
```

Comment:
I'm not sure to truly understand this phrase. The policy says that every pod with both labels `app=bookstore` and `role=api` can be accessed only by pods with label `app=bookstore`.

Pods without the `app=bookstore` label is not affected by this policy. Maybe it's what you had in mind and I didn't get it?

Reply:
Corrected must have messed up the logic somehow